config/initializers/devise.rb

```diff
@@ -142,17 +178,17 @@
 
   # ==> Configuration for :validatable
   # Range for password length.
-  config.password_length = 8..72
+  config.password_length = 6..128
```

Comment:
1) how about we bump this to 8, or better 12?
2) maaaybe this is a good excuse to add that strong passwords into this process, to take advantage of asking folks to change it?

Reply:
per discussion, using this as an opportunity to add strong password changes is a wonderful idea, and we should absolutely add that to the feature branch. I'lll bump back to 8 for now.

```suggestion
  config.password_length = 8..128
```